build/build-config.yml

```diff
@@ -36,6 +36,11 @@ config:
       - work-dir: "core/ifix-zuul"
         image-name: "ifix-zuul"
         dockerfile: "build/maven/Dockerfile"
+  - name: "builds/iFix/core/egov-mdms-service"
```

Comment:
Are you customising mdms for ifix? If not then you do not need build pipeline and code base for mdms in this repo. You can directly deploy the docker image.

Reply:
in mgramserva  there is no mdms pipeline